webapp/app/crops/routes.py

```diff
@@ -26,6 +26,8 @@ def batch_list():
     A time range can be provided as a query parameter. For a batch to be included in the
     page, its weigh-event must have happened in this range.
     """
+    query = queries.crop_types(db.session)
```

Comment:
Are these two lines meant to stay?

Reply:
Good spot!  Indeed that was just for debugging - removed now.